native/src/routes/news/containers/NewsContainer.js

```diff
@@ -190,7 +192,7 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
       navigation,
       route: ownProps.route,
       hasMoreNews: route.hasMoreNews || null,
-      page: route.page || null,
+      page: route.page !== undefined ? route.page : null,
```

Comment:
why this change? Is it possible thate `route.page` is an empty string

Reply:
Page is either a number or undefined. It is undefined if currently a fetch is in progress. This is to avoid sketchy 0 check, i.e. evaluation to null if page is 0.